Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1146,6 +1210,21 @@ namespace Tensile
         if(Debug::Instance().printWinningKernelName())
             std::cout << "Running kernel: " << this->KernelName() << std::endl;
 
+        if(Debug::Instance().printKernelCommonParams())
+        {
+            std::cout << "Kernel (" << this->KernelName() << ") Params:" << std::endl;
+
```

Comment:
Since the kernel name is long, if there is no line wrapping (in a text file), Params will be missed. I suggest:

std::court << " Kernel name: " << this->KernelName() << std::endl;
std::court << " Kernel parameters: " << std::endl;